guides/doc-Release_Notes/topics/foreman-contributors.adoc

```diff
@@ -1,5 +1,5 @@
 We'd like to thank the following people who contributed to the Foreman {{page.version}} release:
 
-Adam Ruzicka, Alex Fisher, Amit Upadhye, Anthony Somerset, Bastian Schmidt, Bernhard Suttner, Chris Roberts, Christian Ruppert, Dirk Götz, Dirk Heinrichs, Dyrkon, Eric D. Helms, Evgeni Golov, Ewoud Kohl van Wijngaarden, Frank Adaemmer, Ian Ballou, Jeremy Lenz, Jonas Trüstedt, Kamil Szubrycht, Karolina Malyjurkova, Leos Stejskal, Lior Keren, Lucy Fu, Lukáš Zapletal, Manuel Laug, Marcel Kühlhorn, Marek Hulán, Maria Agaphontzev, Matt Darcy, Matěj Mudra, Nofar Alfassi, Oleh Fedorenko, Partha Aji, Pat Riehecky, Pavel Moravec, Peter Koprda, Quirin Pamp, Ron Lavi, Samir Jha, Sayan Das, Shim Shtein, William Clark, benjamin-robertson, kobybr, timdeluxe, Štefan Németh
+Adam Hosek, Adam Ruzicka, Alex Fisher, Alexey Masolov, Andrew Teixeira, Antonin Dvorak, Archana Kumari, Areyus, Bastian Schmidt, Bernhard Suttner, Damon Clinkscales, Daniel Alley, Dirk Götz, Eric Cook, Eric D. Helms, Erik Berg, Evgeni Golov, Ewoud Kohl van Wijngaarden, Girija Soni, Gordon Bleux, Griffin Sullivan, Ian Ballou, Jan Klare, Jerome Charaoui, Joshua Hoblitt, Leos Stejskal, Lior Keren, Lucy Fu, Marek Hulán, Maria Agaphontzev, Markus Bucher, Maximilian Kolb, Nagoor Shaik, Nofar Alfassi, Oleh Fedorenko, Pat Riehecky, Peter Bittner, Romain Tartière, Ron Lavi, Samir Jha, Sebastian Bublitz, Tim Meusel, Trey Dockendorf, William Clark, archanaserver, chr1s692, girijaasoni, jc, pkoprda, vijaysawant
```

Comment:
There are a few that we don't properly map here. `girijaasoni` is `Girija Soni`. There are a few others we should expand. At least `archanaserver` and `pkoprda` are Red Hat colleagues. IIRC `vijaysawant` was too.

Reply:
ok let me try running it again but fix the mapping